include/zephyr/bluetooth/l2cap.h

```diff
@@ -545,6 +598,29 @@ int bt_l2cap_chan_disconnect(struct bt_l2cap_chan *chan);
  */
 int bt_l2cap_chan_send(struct bt_l2cap_chan *chan, struct net_buf *buf);
 
+/** @brief Give credits to the remote
+ *
+ *  Only available for channels using @ref bt_l2cap_chan_ops.seg_recv.
```

Comment:
Could mention something like `@kconfig{CONFIG_BT_L2CAP_SEG_RECV} must be enabled for this function`